src/main/java/bowling/domain/Frames.java

```diff
@@ -0,0 +1,56 @@
+package bowling.domain;
+
+import bowling.domain.frame.Frame;import bowling.domain.frame.NormalFrame;import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class Frames {
+    private List<Frame> frames;
+    private Round round;
+
+    public Frames(final Round round) {
+        this.frames = new ArrayList<>();
+        this.round = round;
+    }
+
+    public static Frames newInstance() {
+        return new Frames(Round.from(Round.BOWLING_MIN_ROUND));
+    }
+
+    public void init() {
+        this.frames.add(NormalFrame.newInstance());
+    }
+
+    public boolean isPlayable() {
+        if (frames.size() < Round.BOWLING_MAX_ROUND) {
+            return true;
+        }
+        return !frames.get(Round.BOWLING_MAX_ROUND - 1).isEnd();
+    }
```

Comment:
구현 너무 잘해주셨습니다 👍 
마지막 미션인 만큼 가독성을 위해 조금만 더 `method`를 나눠보시는건 어떨까요?? 😅 

```suggestion
    public boolean isPlayable() {
        if (hasNextRound()) {
            return true;
        }

        return !getFinalRound().isEnd();
    }

    private boolean hasNextRound() {
        return Round.BOWLING_MAX_ROUND > frames.size();
    }

    private Frame getFinalRound() {
        return frames.get(Round.BOWLING_MAX_ROUND - 1);
    }
```

이 부분도 참고만 해주시면 감사하겠습니다 😄